mne/io/edf/tests/test_edf.py

```diff
@@ -556,6 +567,8 @@ def test_read_latin1_annotations(tmp_path):
 
     with pytest.raises(Exception, match="Encountered invalid byte in"):
         _read_annotations_edf(tal_channel)  # default encoding="utf8" fails
+    with pytest.raises(Exception, match="Encountered invalid byte in"):
```

Comment:
```suggestion
    with pytest.raises(
        Exception,
        match="'utf-8' codec can't decode byte 0xe9 in position 1: unexpected end of data",
    ):
```

Reply:
Note that I shortened up the `match`, it's nice to have it on one line and it still conveys the point